### ISSUE TITLE
XLSB Read error

### ISSUE THREAD

**Issue by gustavosimil** [SheetJS/js-xlsx#1044]
I'm trying to read a simple xlsb (Excel binary) file. 
When I try to upload the file, I get the following error: Error: Unexpected record 3072.

I have attached the xlsb file and the SheetJs demo can be used to simulate the issue (http://oss.sheetjs.com/).

[Book1.zip](https://github.com/SheetJS/js-xlsx/files/1827007/Book1.zip)

**Comment by SheetJSDev** [comment]
@gustavosimil  Thank you very much !  Coincidentally we found this issue earlier today and are preparing for the next release.

The `0x0c00` record type is not specified in MS-XLSB but the best guess is that it represents the underspecified `"BrtUid"` record.  We already reported the spec error to Microsoft: https://social.msdn.microsoft.com/Forums/en-US/880eb3f3-395a-4097-ad01-8e0878d0b436/xlsb-brtuid-record-type-not-defined?forum=os_binaryfile .

We will release 0.12.6 soon with the fix.
